Workade/Views&ViewModels/CheckList/CheckListDetail/CheckListBottomSheet/CheckListBottomSheetViewController.swift

```diff
@@ -0,0 +1,119 @@
+        
+        return view
+    }()
+    
+    private let radiusView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .theme.background
+        view.layer.cornerRadius = 30
+        view.layer.maskedCorners = [.layerMinXMaxYCorner, .layerMaxXMaxYCorner]
+        view.clipsToBounds = true
+        view.translatesAutoresizingMaskIntoConstraints = false
+        
+        return view
+    }()
+    
+    private lazy var bottomSheetView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .theme.labelBackground
+        
+        view.addSubview(radiusView)
+        
+        view.translatesAutoresizingMaskIntoConstraints = false
+        
+        return view
+    }()
+    
+    private var bottomSheetViewTopConstraint: NSLayoutConstraint!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let dimmedTap = UITapGestureRecognizer(target: self, action: #selector(dimmedViewTapped(_:)))
+        dimmedView.addGestureRecognizer(dimmedTap)
+        dimmedView.isUserInteractionEnabled = true
+        
+        setupLayout()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        showBottomSheet()
+    }
+
+    private func setupLayout() {
+        view.addSubview(bottomSheetView)
+        view.addSubview(dimmedView)
+        
+        let guide = view.safeAreaLayoutGuide
+        let topConstant: CGFloat = view.safeAreaInsets.bottom + guide.layoutFrame.height
+        bottomSheetViewTopConstraint = bottomSheetView.topAnchor.constraint(equalTo: view.topAnchor, constant: topConstant)
+        
+        NSLayoutConstraint.activate([
+            dimmedView.topAnchor.constraint(equalTo: view.topAnchor),
+            dimmedView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            dimmedView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            dimmedView.bottomAnchor.constraint(equalTo: bottomSheetView.topAnchor)
+        ])
+        
+        NSLayoutConstraint.activate([
+            bottomSheetViewTopConstraint,
+            bottomSheetView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            bottomSheetView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            bottomSheetView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+        
+        NSLayoutConstraint.activate([
+            radiusView.topAnchor.constraint(equalTo: bottomSheetView.topAnchor),
+            radiusView.leadingAnchor.constraint(equalTo: bottomSheetView.leadingAnchor),
+            radiusView.trailingAnchor.constraint(equalTo: bottomSheetView.trailingAnchor),
+            radiusView.heightAnchor.constraint(equalToConstant: 40)
+        ])
+    }
+    
+    private func showBottomSheet() {
+        let safeAreaHeight: CGFloat = view.safeAreaLayoutGuide.layoutFrame.height
+        let bottomPadding: CGFloat = view.safeAreaInsets.bottom
```

Comment:
```suggestion
        let safeAreaHeight = view.safeAreaLayoutGuide.layoutFrame.height
        let bottomPadding = view.safeAreaInsets.bottom
```
[P4] 이 정도는 자료형 선언이 불필요하지 않을까요?